rules/using-digital-asset-manager/rule.md

```diff
@@ -0,0 +1,98 @@
+---
+type: rule
+title: Video Data - Do you use a Digital Asset Manager (DAM) like Adobe Bridge
+  to manage a stock library
+uri: using-digital-asset-manager-for-stock
+authors:
+  - title: William Fardell
+    url: https://ssw.com.au/people/alumni/will/
+    img: https://ssw.com.au/people/static/Will-Greentree-Profile-5b47d1b6e1f91074e10546d28020eb19.jpg
+  - title: SSW RulesGPT 3.5 turbo
```

Comment:
As per our conversation and your question to Uly, please remove this author

Reply:
Done